setup.py

```diff
@@ -29,6 +29,5 @@
         'catkin_pkg',
         'catkin_tools>=0.4.4',
         'catkin_sphinx',
-        'epydoc',
```

Comment:
Removing the dep since `epydoc` doesn't actually exist for Python 3— the last release of it was in 2008. So although this works for now, it's a solution that won't work once the modules being documented can no longer be imported by Python 2.

However, we can probably just sit on it for now and wait for upstream to come up with a solution since they'll need to anyway for ROS N.

Reply:
@tfoote, @dirk-thomas: What are you guys doing about epydoc documentation for ROS 1 stuff once we're in Python 3 land? Is there an existing discussion about this? Seems like the main options are `pdoc`, `pydoctor`, and `sphinx-epytext`, all of which appear to work without having to import the live modules, which eliminates a major annoyance of `epydoc` (as well as resolving the obvious Py3 issue).

Reply:
> Is there an existing discussion about this?

Nothing yet.

Reply:
Okay cool, thanks. Upon further investigation, it looks like pdoc and sphinx-epytext/autodoc do indeed depend on importing the objects, and pydoctor doesn't support Python 3 yet (https://github.com/twisted/pydoctor/issues/96). pdoc looks like it might be abandonware (https://github.com/mitmproxy/pdoc), so it's possible that Sphinx + autodoc is the main viable replacement option.

More discussion: https://github.com/biopython/biopython/issues/906

Reply:
Tried `pdoc` briefly— output is pretty enough, but it doesn't interpret various sequences (`@author`, `@param`), and it renders the entire module to a single file, which I don't love, as rospy's html ended up being 475k.

Reply:
Sphinx isn't ideal either— even with autodoc, you still need a bunch of rST files to define the overall shape of the documentation, so a lot of manual porting work is necessary. Doxygen is the closest to a drop-in replacement, but its default view is very Java-y and class oriented; when the main API of rospy is a bunch of module-level function calls, that's not a helpful view.

Looks like others have put some work into porting epydoc forward, eg: https://github.com/nltk/epydoc/pull/3

Maybe a sane short-term plan would be to just vendor-in that work?

Reply:
The master branch of `pydoctor` should now work fine on Python 3. If not, please report any issues that you run into.

Reply:
pydoctor is working for me, thanks! The template is different from the classic epydoc one, but the content looks great. Will move on porting over catkin_tools_document to use it.